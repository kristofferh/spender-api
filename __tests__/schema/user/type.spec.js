import UserType from "schema/user/type";

describe("User Type", () => {
  it("Should match snapshot", () => {
    expect(UserType.getFields()).toMatchSnapshot();
  });
});
