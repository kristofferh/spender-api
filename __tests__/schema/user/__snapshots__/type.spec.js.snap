// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Type Should match snapshot 1`] = `
Object {
  "email": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "email",
    "type": "String",
  },
  "id": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "id",
    "type": "Int",
  },
  "items": Object {
    "args": Array [
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "after",
        "type": "String",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "first",
        "type": "Int",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "before",
        "type": "String",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "last",
        "type": "Int",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "orderBy",
        "type": "[UserItemsOrderBy]",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "description",
        "type": "String",
      },
    ],
    "description": "The user's items",
    "isDeprecated": false,
    "name": "items",
    "resolve": [Function],
    "type": "UserItemsConnection",
  },
  "tags": Object {
    "args": Array [
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "after",
        "type": "String",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "first",
        "type": "Int",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "before",
        "type": "String",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "last",
        "type": "Int",
      },
      Object {
        "astNode": undefined,
        "defaultValue": undefined,
        "description": null,
        "name": "orderBy",
        "type": "[UserTagsOrderBy]",
      },
    ],
    "description": "The user's tags",
    "isDeprecated": false,
    "name": "tags",
    "resolve": [Function],
    "type": "UserTagsConnection",
  },
  "token": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "token",
    "type": "String",
  },
}
`;
